emission/net/usercache/builtin_usercache_handler.py

```diff
@@ -160,6 +162,21 @@ def storeViewsToCache(self):
         valid_key_list = ["diary/trips-%s"%day for day in day_list_bins.iterkeys()]
         self.delete_obsolete_entries(uc, valid_key_list)
 
+    def storeCommonTripsToCache(self):
+        """ 
+        Determine which set of common trips to send to the usercache. 
+        As of now we will run the pipeline on the full set of data and send that up
+        """
+        tour_model = esdtmpq.get_tour_model(self.user_id)
+        uc = enua.UserCache.getUserCache(self.user_id)
+        logging.debug("Adding common trips for user_id %s" % self.user_id)
+        uc.putDocument("common_trips-%s" % tour_model.user_id, tour_model)
```

Comment:
Two questions here:
- There is only one %s, but there are two fields. Are you sure that this works? Please add the log file lines that show that this works
- why put the user_id into the common trips given that the user cache is user specific? The phone does not actually know the user's user_id so it would be hard to retrieve this document on the phone.